roles/backup/defaults/main.yml

```diff
@@ -20,8 +20,17 @@ clean_backup_on_delete: false
 # Variable to signal that this role is being run as a finalizer
 finalizer_run: false
 
+# Default resource requirements
+backup_resource_requirements:
+  limits:
+    cpu: "500m"
+    memory: "512Mi"
+  requests:
+    cpu: "500m"
```

Comment:
Setting requests this high is likely to cause issues when scheduling the management pod for some folks.  If there aren't enough free resources, it won't be scheduled and the restore will just hang and eventually fail even though the task could be carried out with far fewer resources.  

Similarly, throttling with a limit like that as a default could be problematic for those with much larger databases.  I think it is better to set these defaults high and be on the safe side, and let cluster admins tune it as they see fit.  

I suggest something more like:
```
  backup_resource_requirements:
    limits:
      cpu: "1000m"
      memory: "4096Mi"
    requests:
      cpu: "25m"
      memory: "32Mi"
```